src/main/java/randoop/contract/ObserverEqArray.java

```diff
@@ -134,6 +135,9 @@ public String toCodeString() {
     } else if (value.getClass().getComponentType() == float.class) {
```

Comment:
I suggest to, at the beginning of this class, set a variable to either "org.junit.Assert.assertArrayEquals" or "assertArrayEquals".  Then use it throughout.